### PR TITLE
80330 webjob for cleanup azure tables blobs

### PR DESCRIPTION
**Pull request by mayur1501364**
1. Added cleanup webjob project.
2. Added code to cleanup unnecessary and outdated blobs and table data.
3. Added Unit Test project for webjob
4. Unit test cases covered 
5. Improved Stryker report for webjob

### CODE REVIEW

**Review by jrippington — APPROVED** [review]
Looks good

File: src/UKHO.ERPFacade.Common/IO/Azure/AzureTableReaderWriter.cs [hunk 1]
```diff
@@ -22,6 +22,7 @@ public class AzureTableReaderWriter : IAzureTableReaderWriter
         private const string PriceChangeMasterTableName = "pricechangemaster";
         private const string UnitPriceChangeTableName = "unitpricechangeevents";
         private const string IncompleteStatus = "Incomplete";
+        private const string CompleteStatus = "Complete";
```
Comment (Vishal14583) [inline src/UKHO.ERPFacade.Common/IO/Azure/AzureTableReaderWriter.cs:25]: Can we have enum for this please?

**Review by Vishal14583 — CHANGES_REQUESTED** [review]
Minor comment and one confilct to resovle otherwise looks good

File: src/UKHO.ERPFacade.CleanUp.WebJob/CleanUpWebjob.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using UKHO.ERPFacade.CleanUp.WebJob.Services;
+using UKHO.ERPFacade.Common.Configuration;
```
Comment (Shirin14926) [inline src/UKHO.ERPFacade.CleanUp.WebJob/CleanUpWebjob.cs:10]: Remove unused namespaces

File: src/UKHO.ERPFacade.Common/IO/Azure/AzureTableReaderWriter.cs
Comment (Shirin14926) [inline src/UKHO.ERPFacade.Common/IO/Azure/AzureTableReaderWriter.cs:1]: Also check unused namespaces in this file

**Review by Shirin14926 — COMMENTED** [review]
Minor comment please

File: src/UKHO.ERPFacade.Common/IO/Azure/AzureTableReaderWriter.cs (hunk 1 above)
Comment (mayur1501364) [reply src/UKHO.ERPFacade.Common/IO/Azure/AzureTableReaderWriter.cs]: Added enum

File: src/UKHO.ERPFacade.CleanUp.WebJob/CleanUpWebjob.cs (hunk 2 above)
Comment (mayur1501364) [reply src/UKHO.ERPFacade.CleanUp.WebJob/CleanUpWebjob.cs]: removed unused namespaces

File: src/UKHO.ERPFacade.Common/IO/Azure/AzureTableReaderWriter.cs
Comment (mayur1501364) [reply src/UKHO.ERPFacade.Common/IO/Azure/AzureTableReaderWriter.cs:1]: Removed unused namespaces

**Review by Vishal14583 — APPROVED** [review]
LGTM

**Pull request merged**